fs.test

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Are the files on this level dupe of what's inside `test/data` dir? I believe you meant to commit just the `test/data` correct?

Reply:
Yes, only test/data